content/practice/establish-shared-principles.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Establish Shared Principles
+subtitle: Principles Over Practices
+date: 2019-05-22T16:05:41.031Z
+authors:
+  - tenfourty
+area: discovery-loop-outcomes
+perspectives:
+  - start-with-why
```

Comment:
This perspective does not exist yet. It is currently a practice. I would suggest to remove this line for now or make "start-with-why" a perspective.